src/m365/external/commands/connection/connection-add.ts

```diff
@@ -3,6 +3,7 @@ import { Logger } from '../../../../cli/Logger.js';
 import GlobalOptions from '../../../../GlobalOptions.js';
 import request from '../../../../request.js';
 import GraphCommand from '../../../base/GraphCommand.js';
+import searchCommands from '../../commands.js';
```

Comment:
You're importing the same thing twice here. That's not really necessary, just use the existing commands import